tests/parsers/appbio_quantstudio_designandanalysis/appbio_quantstudio_designandanalysis_parser_test.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from allotropy.parser_factory import Vendor
+from tests.parsers.test_utils import from_file, validate_contents
+
+OUTPUT_FILES = (
+    "appbio_quantstudio_designandanalysis_QS1_Standard_Curve_example01",
+    "appbio_quantstudio_designandanalysis_QS3_Relative_Quantification_example02",
+    "appbio_quantstudio_designandanalysis_QS5_Standard_Curve_4Plex_example03",
+    "appbio_quantstudio_designandanalysis_QS6_Standard_Curve_example04",
+    "appbio_quantstudio_designandanalysis_QS6Pro_Standard_Curve_example05",
+    "appbio_quantstudio_designandanalysis_QS7_Standard_Curve_example06",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Genotyping_example07",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Multiplex_example08",
+    "appbio_quantstudio_designandanalysis_QS7Pro_PCR_with_Melt_example09",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Presence_and_Absence_example10",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Relative_Quantification_example11",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Relative_Quantification_Biogroup_example12",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Relative_Standard_Curve_example13",
+    "appbio_quantstudio_designandanalysis_QS7Pro_Standard_Curve_example14",
+    ## "appbio_quantstudio_designandanalysis_QS7Pro_Standard_Curve_TAC_example15",
```

Comment:
Add comment on why these tests are excluded, or remove?

Reply:
Those are files not supported in this version, but they should be supported in the next one